src/__tests__/nav.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import {cleanup, fireEvent, render, screen} from '@testing-library/react';
+import Toggle from '../components/CustomInputs/Toggle';
+import Nav from '../components/Nav';
+
+describe('Nav tests', () => {
```

Comment:
You could [write fewer, longer tests](https://kentcdodds.com/blog/write-fewer-longer-tests).